lib/ex_unit/lib/ex_unit.ex

```diff
@@ -65,12 +65,18 @@ defmodule ExUnit do
     1. Passed (also represented by `nil`)
     2. Failed
     3. Skipped (via @tag :skip)
```

Comment:
Minor nitpick: do you mind adding backticks here? `@tag :skip`, `:exclude`, `setup_all/1,2` 🙂